### ISSUE TITLE
KeyError in DrugCentral_Conversion rule

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX-KG2#109]

**Comment by saramsey** [comment]
See original error message in #104 
Seems to be no logging for the `DrugCentral_Conversion` rule; in `Snakefile-conversion`:
```
rule DrugCentral_Conversion:
    input:
        real = config['DRUGCENTRAL_INPUT_FILE'],
        validation = config['VALIDATION_PLACEHOLDER']
    output:
        config['DRUGCENTRAL_OUTPUT_FILE']
    shell:
        config['VENV_DIR'] + "/bin/python3 -u " + config['CODE_DIR'] + "/drugcentral_json_to_kg_json.py " + config['TEST_ARG'] + " {input.real} {output}"
```

**Comment by saramsey** [comment]
Here is the structure of the `version` dictionary in the JSON blob in `kg2-build/drugcentral/drugcentral_psql_json.json`:
```
>>> x['version'][0]
{'version as version_number': '48', 'dtime': '2020-09-15 22:46:50.315'}
```
so there is no key `version` inside that dictionary, only a key `version as version_number`.

**Comment by saramsey** [comment]
fixed in bbd6270
